app/views/admin/users/show/profile/actions/_export.html.erb

```diff
@@ -1,19 +1,15 @@
 <div id="export-data">
   <div class="flex flex-col gap-4">
     <p>
-      You can export a user's data here. Currently we only support exporting a user's posts and comments.
+      Choose betweem sending the exported data to either your Admin contact email, <%= ForemInstance.email %>,
+      or to <%= @user.name %>'s <a href="<%= @user.email %>">user email</a> directly.
```

Comment:
```suggestion
      or to <%= @user.name %>'s <a href="mailto:<%= @user.email %>">user email</a> directly.
```

At the moment this is directing to e.g. `http://localhost:3000/admin/users/lidia+collier@oberbrunner.info`, which doesn't exist

Reply:
Great catch; thanks!!